src/Microsoft.Data/PrimitiveDataFrameColumn.cs

```diff
@@ -278,7 +284,7 @@ public override bool IsNumericColumn()
             return ret;
         }
 
-        public override DataFrameColumn FillNulls(object value, bool inPlace = false)
+        public new PrimitiveDataFrameColumn<T> FillNulls(object value, bool inPlace = false)
```

Comment:
You could change this to:

```C#
public new PrimitiveDataFrameColumn<T> FillNulls(T value, bool inPlace = false)
```

And then you wouldn't need to change the type below.

Reply:
But that is not equivalent to what the code is doing now right? `Convert.ChangeType` will convert `value` to `T`. Let's say I have a `PrimitiveDataFrameColumn<double> doubleCol`. If I called `doubleCol.FillNulls(0.2f)`, `Convert.ChangeType` will convert the float to a double.

Reply:
`float` has an implicit conversion to `double. So the C# compiler will do the conversion.

It is not equivalent, but it is better. That's why we have strongly typed APIs. If you want the weakly typed `FillNulls` call the base class's `FillNulls` that takes an `object`.